drivers/espi/espi_mchp_xec_v2.c

```diff
@@ -1064,6 +1064,19 @@ static void notify_host_warning(const struct device *dev,
 	}
 }
 
+static void notify_vw_status(const struct device *dev,
```

Comment:
It seems like the implementation here is the same as notify_system_state, maybe worth consolidating the body into a common inline? I get the intent given the function name may be different and the alias might be nice.